com.unity.multiplayer.mlapi/Tests/Editor/NetworkBehaviourTests.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using NUnit.Framework;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace MLAPI.EditorTests
+{
+    public class NetworkBehaviourTests
+    {
+        [Test]
+        public void HasNetworkObjectTest()
+        {
+            var gameObject = new GameObject(nameof(NetworkObject));
+            var networkBehaviour = gameObject.AddComponent<EmptyNetworkBehaviour>();
+            
+            Assert.That(networkBehaviour.HasNetworkObject, Is.False);
+
+            var networkObject = gameObject.AddComponent<NetworkObject>();
+            
+            Assert.That(networkBehaviour.HasNetworkObject, Is.True);
+            
+            Object.DestroyImmediate(networkObject);
+            
+            Assert.That(networkBehaviour.HasNetworkObject, Is.False);
+        }
+        
+        [Test]
+        public void AccessNetworkObjectTest()
+        {
+            var gameObject = new GameObject(nameof(NetworkObject));
```

Comment:
```suggestion
            var gameObject = new GameObject(nameof(AccessNetworkObjectTest));
```
minor: as suggested above, more specific naming instead of generic